src/input/input_json.rs

```diff
@@ -172,6 +173,18 @@ impl<'a> Input<'a> for JsonInput {
         }
     }
 
+    fn strict_decimal(&'a self, decimal_type: &'a PyType) -> ValResult<&'a PyAny> {
+        let py = decimal_type.py();
+        match self {
+            JsonInput::Float(f) => create_decimal(PyString::new(py, &f.to_string()), self, decimal_type),
```

Comment:
Is there a way for us to make sure that when parsing numeric JSON into a decimal it is treated as a string rather than loaded into an f64 or whatever? I think @samuelcolvin said he did something like this in speedate

Reply:
(Maybe it's already doing that, not sure, but it's probably worth checking that we load many-digit JSON numbers into decimal properly)